content/patreon.md

```diff
@@ -3,7 +3,7 @@
 [//]: # (DESCRIPTION:Extra perks of becoming a sponsor)
 [//]: # (TAGS:regions,areas,worldguard,world,region,media,accounts)
 
-OpenAudioMc has always been a hobby project that I ran in my spare time ever since it launched <b>5 years</b> ago.
+OpenAudioMc has always been a hobby project that I ran in my spare time ever since it launched `5 years` ago.
```

Comment:
whats wrong with bold?

Reply:
Lets say, all pages  make use of the blue shiny highlight, this is the only page you used bold. This was a suggestion from IOByte, but feel free to change it back if you would like to